active/NavigationView/NavigationViewTemplateSettings/PaneTitle.md

```diff
@@ -0,0 +1,47 @@
+# Background
+PaneTitle is added to NavigationView in RS4, but it's a String. Since RS5, we introduced PaneHeader(UIElement) to deprecate PaneTitle. If both PaneTitle and PaneHeader are provided by customer, PaneHeader is used.
+
+This API update addes PaneTitle [Template-Settings](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/template-settings-classes) to the NavigationView control.
+
+# Description
+Provides calculated PaneTitle that can be referenced as TemplatedParent sources when defining templates for a NavigatiionView control. Not intended for general use.
+
+# Examples
+
+```
+[WUXC_VERSION_RS5]

```

Comment:
For API review we don't include these attributes because they don't mean anything outside our team.
```suggestion
```